source/ada/lsp-ada_contexts.ads

```diff
@@ -59,6 +59,10 @@ package LSP.Ada_Contexts is
    function Id (Self : Context) return LSP.Types.LSP_String;
    --  Return unique identifier of the context.
 
+   function Analysis_Units
+     (Self : Context) return Libadalang.Analysis.Analysis_Unit_Array;
+   --  Return the analysis units for all Ada sources known to this context
+
```

Comment:
Can we move `Get_Importing_Units` into `LSP.Ada_Contexts package` instead of making `Analysis_Units` visible?

Reply:
Ok